dateparser/__init__.py

```diff
@@ -54,4 +55,4 @@ def parse(date_string, date_formats=None, languages=None, locales=None, region=N
     data = parser.get_date_data(date_string, date_formats)
 
     if data:
-        return data['date_obj']
+        return data['date_obj']
```

Comment:
Could you revert the changes for this field? You should remove the line before `_default_parser = DateDataParser()` and add the empty line at the end of the file